pkg/cloudprovider/aws/instancetype.go

```diff
@@ -117,29 +117,31 @@ func (i *InstanceType) computeRequirements() scheduling.Requirements {
 		v1.LabelArchStable:         sets.NewSet(i.architecture()),
 		v1.LabelOSStable:           sets.NewSet(v1alpha5.OperatingSystemLinux),
 		v1.LabelTopologyZone:       sets.NewSet(lo.Map(i.Offerings(), func(o cloudprovider.Offering, _ int) string { return o.Zone })...),
+		// Well Known to Karpenter
 		v1alpha5.LabelCapacityType: sets.NewSet(lo.Map(i.Offerings(), func(o cloudprovider.Offering, _ int) string { return o.CapacityType })...),
-		// Resources
-		v1alpha1.InstanceCPULabelKey:    sets.NewSet(fmt.Sprint(aws.Int64Value(i.VCpuInfo.DefaultVCpus))),
-		v1alpha1.InstanceMemoryLabelKey: sets.NewSet(fmt.Sprint(aws.Int64Value(i.MemoryInfo.SizeInMiB))),
+		// Well Known to AWS
+		v1alpha1.InstanceCPULabelKey:             sets.NewSet(fmt.Sprint(aws.Int64Value(i.VCpuInfo.DefaultVCpus))),
```

Comment:
would be nice for these to match the upstream labels and our capacity type label:

```
v1alpha1.LabelInstanceCPU
v1alpha1.LabelInstanceMemory
...
```

Reply:
Great suggestion!